### PR TITLE
W-15491135 Add needed step of adding scope to root org

### PR DESCRIPTION
**Pull request by marinasasso**
1. New needed step to add the "Application Creator" scope to the Root Organzation when creating a new connected app. (This was feedback from Mina Kim in the #ms-docs-feedback channel, verified with eng).
2. Removed unnecessary screen shot & associated callouts.
3. Changed casing to use lowercase "connected app" everywhere we're not specifically talking about the Connected App feature.
4. Added the word feature a couple places where we were saying "Do blah using Connected Apps", we now say "Do blah using the Connected App Feature".

### CODE REVIEW

File: modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc [hunk 1]
```diff
@@ -19,12 +19,7 @@ When you create the Connected App to manage client applications, ensure that you
 
 . Select *App acts on its own behalf (client credentials)* for *Type*.
 . Add the *Application Creator* scope.
-
-image::connected-app-client-apps.png[Screenshot calling out the location of App acts on its own behalf (client credentials) and Application Creator]
-
-[calloutlist]
-. *App acts on its own behalf (client credentials)* type
-. *Add Scopes* button to add *Application Creator* scope
+. Select the root organization as the *Business Group* as you add this scope.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]: The "as...as" structure can be ambiguous. What's the meaning of this sentence?:

-  Select the root organization as the *Business Group* when you add this scope.
-  Select the root organization as the *Business Group* if you add this scope.
-  Select *Business Group* in the root organization when/if you add this scope.

Comment (marinasasso) [reply modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]: Good point! Now reworded to be more clear :)

File: modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc [hunk 2]
```diff
@@ -13,18 +13,13 @@ Connected Apps help users delegate their access without sharing sensitive creden
 
 == Creating a Connected App
 
-To create a Connected App, see xref:access-management::connected-apps-developers.adoc[].
+To create a Connected App, see xref:access-management::connected-apps-developers.adoc[]
```
Comment (hannanelson) [inline modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]: Why are you removing the period?

Comment (hannanelson) [inline modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]:
```suggestion
To create a connected app, see xref:access-management::connected-apps-developers.adoc[].
```

Comment (hannanelson) [reply modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]: We use lowercase when referring to the thingie generically; init cap for the feature name.

**Review by hannanelson — DISMISSED** [review]

**Comment by marinasasso** [comment]
https://github.com/mulesoft/docs-api-manager/pull/557

File: modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc [hunk 3]
```diff
@@ -1,50 +1,45 @@
...
+For automation use cases, you can use the Connected App feature to create and manage client applications.
 
-Connected Apps help users delegate their access without sharing sensitive credentials or giving full control of their applications to third parties. Actions taken by connected apps are audited, and users can revoke access at any time.
+Connected apps help users delegate their access without sharing sensitive credentials or giving full control of their applications to third parties. Actions taken by connected apps are audited, and users can revoke access at any time.
 
 
 == Creating a Connected App
 
-To create a Connected App, see xref:access-management::connected-apps-developers.adoc[].
+To create a connected app, see xref:access-management::connected-apps-developers.adoc[]
```
Comment (hannanelson) [inline modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]: I still think this needs terminating punctuation...

File: modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc [hunk 4]
```diff
@@ -1,50 +1,45 @@
...
-The Connected App can manage the client application it created, and can perform the following actions:
+The connected app can manage the client application it created, and can perform the following actions:
 
 * Add users as client application owners
 * Request access to an API instance
 * Request a tier change
 * Reset a client applications secret
 
-You cannot add a Connected App as an owner of a client application. Connected Apps can only manage the client applications they create. 
+You cannot add a connected app as an owner of a client application. connected apps can only manage the client applications they create.
```
Comment (hannanelson) [inline modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]:
```suggestion
You cannot add a connected app as an owner of a client application. Connected apps can only manage the client applications they create. 
```

File: modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc (hunk 3 above)
Comment (marinasasso) [reply modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]: So right!

File: modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc (hunk 2 above)
Comment (marinasasso) [reply modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc]: Sorry! I will get better at noticing *all* the comments :)

File: modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc [hunk 5]
```diff
@@ -102,4 +97,4 @@ curl --X POST 'https://anypoint.mulesoft.com/apiplatform/repository/v2/organizat
 
 * xref:exchange::to-request-access.adoc[Request Access to API Instances from Exchange]
 * xref:remove-client-app-latest-task.adoc[Remove API Instance Contracts]
-* xref:access-client-app-id-task.adoc[Obtain Client ID and Client Secret Credentials of a Client Application]
\ No newline at end of file
+* xref:access-client-app-id-task.adoc[Obtain Client ID and Client Secret Credentials of a Client Application]
```
Comment (hannanelson) [inline modules/ROOT/pages/manage-client-apps-connected-apps-concept.adoc:100]: Tip: if you leave the [] blank, antora will automagically fill it in with the current heading text (and you won't need to manually update it if the heading changes).

**Review by hannanelson — APPROVED** [review]

**Pull request merged**